README.rst

```diff
@@ -57,19 +57,19 @@ Installation
 
 PyDy has hard dependencies on the following software:
```

Comment:
Maybe do this:

PyDy has hard dependencies on the following software\ [#]_:

.. [#] We only test PyDy with these minimum dependencies, but previous versions of NumPy and SciPy are likely to work too.